account_statement_import_txt_xlsx/models/account_statement_import_sheet_mapping.py

```diff
@@ -61,6 +62,23 @@ class AccountStatementImportSheetMapping(models.Model):
         help="When this occurs please indicate the column number in the Columns section "
         "instead of the column name, considering that the first column is 0",
     )
+    skip_empty_lines = fields.Boolean(
+        "Skip Empty Lines",
+        default=False,
+        help="Allows to skip empty lines",
+    )
+    offset_column = fields.Integer(
+        "Offset Column",
+        default=0,
+        required=True,
```

Comment:
Why do you need `required`? Especially if the field is optionally hidden

Reply:
@alexey-pelykh 
Thanks for pointing that out. I forgot about it from an earlier iteration